modules/failure-handler/src/main/java/org/apache/ignite/internal/failure/handlers/configuration/NoOpFailureHandlerConfigurationSchema.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.failure.handlers.configuration;
+
+import org.apache.ignite.configuration.annotation.PolymorphicConfigInstance;
+
+/**
+ * Configuration schema for {@link org.apache.ignite.internal.failure.handlers.NoOpFailureHandler}.
+ */
+@PolymorphicConfigInstance(NoOpFailureHandlerConfigurationSchema.TYPE)
+public class NoOpFailureHandlerConfigurationSchema extends FailureHandlerConfigurationSchema {
+    /** Failure handler type name. */
+    public static final String TYPE = "noOp";
```

Comment:
Should it be a "noop" instead? Why did you capitalize the O? It's a commonly known word

Reply:
changed to `noop`